tests/integration/frameworks/wordpress/test_wordpress_apply_filters.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should properly instrument Wordpress apply_filters hooks.
+*/
+
+/*SKIPIF
+<?php
+*/
+
+/*INI
+newrelic.framework = wordpress
+*/
+
+/*EXPECT
+f: string1
+g: string2
+*/
+
+/*EXPECT_METRICS
+[
+  "?? agent run id",
+  "?? start time",
+  "?? stop time",
+  [
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/all"},      [1, "??", "??", "??", "??", "??"]],
+    [{"name": "DurationByCaller/Unknown/Unknown/Unknown/Unknown/allOther"}, [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Framework/WordPress/Hook/f"},                          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Framework/WordPress/Hook/g"},                          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/all"},                                [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransaction/php__FILE__"},                        [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime"},                           [1, "??", "??", "??", "??", "??"]],
+    [{"name": "OtherTransactionTotalTime/php__FILE__"},               [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Forwarding/PHP/enabled"},       [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/Logging/Metrics/PHP/enabled"},          [1, "??", "??", "??", "??", "??"]],
+    [{"name": "Supportability/framework/WordPress/forced"},           [1, "??", "??", "??", "??", "??"]]
+  ]
+]
+*/
+
+// Simple mock of wordpress's apply_filter()
+function apply_filters($tag, ...$args) {
+    call_user_func_array($tag, $args);
+}
+
+function g($str) {
+    echo "g: ";
+    echo $str;
+    echo "\n";
+}
+
+function f($str) {
+    echo "f: ";
+    echo $str;
+    echo "\n";
+    apply_filters("g", "string2");
+}
+
```

Comment:
Is it possible to nest it somehow so we now we are building the internal stack then throw an exception?

Reply:
added in [79c7ee8](https://github.com/newrelic/newrelic-php-agent/pull/559/commits/79c7ee84f4d33549eb345c94afaea0a58874d33d)

Reply:
added in [6cd6234](https://github.com/newrelic/newrelic-php-agent/pull/559/commits/6cd62348111bb4a5a27ac059a782f3e8961f13e2)